src/lib/viewers/controls/color-picker/__tests__/ColorPickerControl-test.tsx

```diff
@@ -76,6 +76,21 @@ describe('ColorPickerControl', () => {
             expect(getColorPickerPalette(wrapper).hasClass('bp-is-open')).toBe(true);
         });
 
+        test('should close the palette when focus is outside palette', () => {
+            const wrapper = getWrapper();
+            const toggleButton = getToggleButton(wrapper);
+            const divNode = document.createElement('div');
```

Comment:
We would typically refer to this as `divEl` or `divElement`, since we know it's an Element object rather than a generic Node.